project-canvas/src/components/login/jira-cloud/JiraCloudLogin.tsx

```diff
@@ -16,7 +18,7 @@ export function JiraCloudLogin({
         <Loader size="xl" />
       </Center>
       <Button variant="outline" fullWidth color="dark" onClick={goBack}>
-        Go Back
+        {t("Common.btn-goBack")}
```

Comment:
Accessing properties through strings is a recipe for disaster. 
As we are using TypeScript, we can luckily avoid that. Here is how to integrate TypeScript with i18next: 
https://www.i18next.com/overview/typescript
Here is also a cli that generates the type files:
https://www.npmjs.com/package/i18next-typescript
Make sure it's good before adding it (2 years old and very few downloads)